### DISCUSSION TITLE
ground truth images of test data for multi-coil knee dataset

### DISCUSSION THREAD

**Discussion by King-pand** [facebookresearch/fastMRI#337] [General]
In the multi-coil knee test data set, I did not find his ground truth image. How can I obtain the test set data with ground truth images?

**Comment by mmuckley** [comment]
Hello @King-pand, the ground truth data for the knee test set has not been released. it is now controlled by NYU directly, and they usually don't respond to questions on GitHub. You could contact them at [fastmri@med.nyu.edu](https://fastmri.med.nyu.edu/fastmri@med.nyu.edu).

**Reply by King-pand** [reply]
Thank you very much for your reply.

**Comment by hu2274898** [comment]
hii do u solve this problem？i find that if i want to evaluate,i must have the ground truth of test dataset which is not shown😭😭
